lib/mayaUsd/fileio/utils/meshWriteUtils.cpp

```diff
@@ -36,14 +36,178 @@
 #include <pxr/usd/usdGeom/tokens.h>
 #include <pxr/usd/usdUtils/pipeline.h>
 
+#include <maya/MBoundingBox.h>
+#include <maya/MFnAttribute.h>
+#include <maya/MFnMesh.h>
 #include <maya/MFnSet.h>
 #include <maya/MGlobal.h>
 #include <maya/MIntArray.h>
+#include <maya/MItDependencyGraph.h>
 #include <maya/MItMeshFaceVertex.h>
 #include <maya/MPlug.h>
 #include <maya/MPlugArray.h>
+#include <maya/MPoint.h>
 #include <maya/MStatus.h>
 #include <maya/MUintArray.h>
+#include <maya/MVector.h>
+
+#define MAYA_ATTR_NAME_INMESH "inMesh"
```

Comment:
I mostly rely on macros for Maya attribute names since some areas require `const char *` or `MString`. I'm more inclined to something like

```static constexpr char kMayaAttrNameInMesh[] = "inMesh";```
though, rather than an anonymous namespace. Is there a reason that it helps? I'm actually not sure.

Reply:
Cool, yeah, a `const char[]` sounds good to me.

For the anonymous namespace, I don't think it really makes a difference in this case. I tend to lean towards that over using `static` just in case at some point we end up wanting to add an internal class or something, in which case `static` doesn't work and we'd already have a namespace for it to go into.

The Cpp core guidelines have an item on using anonymous namespaces, but the justification is admittedly pretty thin:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities